src/server/lib/experiments.ts

```diff
@@ -0,0 +1,44 @@
+import { Request } from 'express';
+import { parse, stringify } from 'query-string';
+import ms from 'ms';
+
+import { ResponseWithRequestState } from '@/server/models/Express';
+
+const RAN_EXPERIMENTS_COOKIE_NAME = 'ranExperiments';
```

Comment:
in general we preface cookies with `GU_`
as a possible suggestion maybe something like:
```suggestion
const RAN_EXPERIMENTS_COOKIE_NAME = 'GU_gateway_experiments';
```

Reply:
Have taken your suggestion.